### PR TITLE
update-tr-language

### PR DESCRIPTION
**Pull request by mikropsoft**

### CODE REVIEW

File: app/src/main/res/values-tr/strings.xml [hunk 1]
```diff
@@ -411,61 +413,83 @@
...
-    <string name="site_legal_info_name">%1$s yasal bilgi</string>
+    <string name="report_person">%1$s\'i raporla</string>
+    <string name="block_person">%1$s\'i engelle</string>
+    <string name="undistinguish_comment">Yorumu sıradanlaştır</string>
+    <string name="distinguish_comment">Yorumu öne çıkar</string>
+    <string name="site_info_name">%1$s bilgisi</string>
+    <string name="site_legal_info_name">%1$s yasal bilgisi</string>
     <string name="moderators">Moderatörler</string>
     <string name="admins">Yöneticiler</string>
-</resources>
```
Comment (MV-GH) [inline app/src/main/res/values-tr/strings.xml:471]: Put this back

**Review by MV-GH — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-tr/strings.xml [hunk 2]
```diff
@@ -411,61 +412,84 @@
...
+    <string name="unable_to_retrieve">Alınamıyor</string>
+    <string name="blocks">Engellemeler</string>
+    <string name="unblock">Engeli kaldır</string>
+    <string name="blocked_users">Engellenen kullanıcılar</string>
+    <string name="blocked_communities">Engellenen topluluklar</string>
+    <string name="blocked_instances">Engellenen örnekler</string>
+    <string name="you_have_no_blocked_users">Engellenen kullanıcınız yok</string>
+    <string name="you_have_no_blocked_communities">Engellenen topluluğunuz yok</string>
+    <string name="you_have_no_blocked_instances">Engellenen örneğiniz yok</string>
+<resources>
```
Comment (MV-GH) [inline app/src/main/res/values-tr/strings.xml:495]: Its `</resources>`

**Review by MV-GH — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-tr/strings.xml (hunk 2 above)
Comment (mikropsoft) [reply app/src/main/res/values-tr/strings.xml:495]: Isn't that how it should be anyway? End code

Comment (MV-GH) [reply app/src/main/res/values-tr/strings.xml:495]: ? you have a opening tag, you need a closing tag.

Comment (mikropsoft) [reply app/src/main/res/values-tr/strings.xml]: > ? you have a opening tag, you need a closing tag.

I forgot the small detail :)

**Review by MV-GH — APPROVED** [review]

**Pull request merged**